### ISSUE TITLE
Un-nerf redcrystal a bit 

### ISSUE THREAD

**Issue by Alastors** [GTNewHorizons/GT-New-Horizons-Modpack#7522]
#### Which modpack version are you using?
2.1.0.3
#
#### If in multiplayer; On which server does this happen? 
Mine
#
#### What did you try to do, and what did you expect to happen?
craft redcrystal like normal
#
#### What happened instead? (Attach screenshots if needed)
crafted redcrystal at a wayyyyy higher price
#
#### What do you suggest instead/what changes do you propose?
The nerf for redcrystal was wayyyyyyyy to high, id commend if you want to nerf it a lil bit to up the Vitreus cost to maybe 4, tbh viterus was kind of annoying to make in general just for redcrystal, the praecantatio kind of makes sense in the context but the cost is way too high, i'd suggest reducing that down to 1 praecantatio at most, Gelum doesnt make sense at all in the context of redcrystal since 1) there isn't enough blocks that produce it to justify it and 2) you wouldnt be freezing redcrystal which is what the context of gelum is, you are crystalizing it, thats what the vitreus cost was for.

My point is that through this you are further alienating what little use thaumcraft gets in this modpack as since you've probably noticed a lot of people in the pack see the mod as stupid and pointless and but nerfing one of the few features used often such as redcrystal this hard reduces the use an already niche mod will see.

 As I said above if you wish to nerf it up the viterus cost to 4 and maybe add a single point praecantatio but why would you nerf a single line of redstone that hard to require 5 aspects of 3 different aspects for what I will stress hard, *a single meter of redstone* 

I would also like to put the cost of the advanced crystals should be nerfed yes but, I propose that instead of using 8 slivers (which isnt expensive its just kind of absurd) that instead it uses a single shard of the respective type.

Also the 4 pure nether quartz for the resonating redcrystal doesnt make sense in the context of the item, you arent calling upon anything that you would need quartz for, if anything if you really want to nerf the original recipe you should instead require infusion requiring like 5 potentia, 2 machina, and 3 viterus, and require 2 aer shards and either 1 ender pearl and another redstone dust or just an additional 2 redstone dust, or even 2 redcrystal.

All in all my recommendation is to not think of thaumcraft nerfs like you would gregtech machine nerfs, like think of the context of the magic and whether what you are doing would make sense in the context of that. (such as adding gelum to the redcrystal recipe, it isnt ice why would it require the ice aspect)

tldr: switch the slivers from advanced redcrystal to shards and dont mess with the original redcrystal recipe, I assure you it isnt any more op than redstone dust, genuinely at times it can be more useless than the dust itself.

**Comment by Alastors** [comment]
I will add that the original recipe for resonating does require quartz but pure quartz is still certainly absurd to make for a terrible 15 block bypass that has to face the other one so you have to make at least 2 channels if you even want to do anything with it. plus it doesnt work on the top or bottom so you have to use the sides of blocks, all in all resonating redcrystal is certainly not as good as that recipe would mark

**Comment by Alastors** [comment]
I played around with it for two hours yesterday, the two redcrystals have to be facing eachother like if one is on top the other has to be on bottom. what I meant by that was they don't transmit to redstone/crystal that is on top of the block they are on if they are on the side so dont @ me

**Comment by Alastors** [comment]
also it wasnt an increase from 4 it was an increase from 1

**Comment by KiloJoel** [comment]
I believe what is meant is
This works:
BR <-14 block gap->RB

This doesn't:
RB <-14 block gap->RB

**Comment by Alastors** [comment]
no no no what I mean is 


R                                  R
BR <-----Gap----->RB

The two redcrytals on top of the blocks do get power for some reason

**Comment by yukieiji** [comment]
RedCrystal is almost the same as RedStone on its own(It has the advantage of being able to switch the connection freely), but RemoteComparator, DenseRedCrystal, and Mercurial RedCrystal(also ResonatingRedCrystal), which are made from it, can be controlled with a single block, which is difficult to do with other mods. As such, I don't think it should be un-nerfed.

**Comment by Lonzo34** [comment]
Alastors > Huh. That's really weird. Only thing I can think of is the connection between the redcrystal on the top and the resonating redcrystal on the side got turned off. Otherwise it's a bug.

Yukieiji > I'm not quite sure what you mean by 'controlled with a single block'. Those are all useful single blocks (with the exception of mercurial), but they're not being controlled by anything.

Regardless, I can see why redcrystal got nerfed in the first place- it was too cheap when compared to red alloy wire (and by extension, redstone conduit)- but with it being behind magic, I think the nerf was too harsh. The main issues I take with the change are the costs of 5 (Just make it 4- fits with jars better), the addition of gelum, and the addition of praecantatio. (In that order)

I don't think un-nerfing is called for. Redcrystal is more powerful than the redstone it's made of (as it should be), but was too cheap compared to red alloy wire and redstone conduit. Its variants offer unique ways to control signal level (some of which are, yes, hard to do with other mods, but *that's the frikken point*), but I haven't found it enough for more complex logic. Nerfing redcrystal as a way to proxy-nerf its variants is like nerfing red alloy wire to nerf all of project red's gates and cells and whatnot- it doesn't make sense to me.

Redcrystal should be less expensive than red alloy wire, but more expensive than it was. Therefore, I propose changing the cost to 2 praecantatio and 4 vitreus, or even keep one gelum if you *really* want to add that scarcely-used aspect.


As for the assertion that mercurial redcrystal is in any way useful, I argue that it is NOT. The direct alternative is WR-CBE Logic's wireless redstone, which does the job better and more cheaply. The recipes my seem the other way around, what with wireless redstone needing tungstensteel (and therefore being gated behind mars), however mercurial redstone require magic mirrors, which require enderium, which needs an ebf with -wait for it- *tungstensteel* coils! (Also, the requirement of magic mirrors means mercurial has a larger footprint than wireless, and disqualifies it from being a single-block contraption)

**Comment by Alastors** [comment]
I would argue gelum by all means should not be part of the recipe, as I still argue it doesnt make sense in the context and would argue yall should use the machine aspect (machina or whatever) if you really want a more niche aspect tbh since as of now youd need to automate redcrystal production just to make it.
